ghost/i18n/locales/de/portal.json

```diff
@@ -160,7 +160,7 @@
     "Welcome to {{siteTitle}}": "Willkommen bei {{siteTitle}}",
     "When an inbox fails to accept an email it is commonly called a bounce. In many cases, this can be temporary. However, in some cases, a bounced email can be returned as a permanent failure when an email address is invalid or non-existent.": "Wenn ein Posteingang eine E-Mail nicht annimmt, spricht man von einem Bounce. In vielen Fällen kann dies vorübergehend sein (Soft Bounce). In einigen Fällen kann eine E-Mail jedoch als dauerhafter Fehler zurückgeschickt werden, wenn eine E-Mail-Adresse ungültig oder nicht vorhanden ist (Hard Bounce).",
     "Why has my email been disabled?": "Warum wurde meine E-Mail deaktiviert?",
-    "year": "",
+    "year": "jahr",
```

Comment:
Also a noun: `Jahr`

Reply:
Hey @lsinger: Some context on month and year - these are the /month and /year strings displayed with a paid plan.  Does that change your feelings about capitalization?  (Maybe it doesn't, but in American English, that would look weird.)

![image](https://github.com/user-attachments/assets/816242a0-caac-496b-97b3-522cb44682fe)

Reply:
@cathysarisky thanks for checking back! Yes, that still needs to be capitalized in German -- every noun needs to be. 😅 I've opened a PR with my notes from this one and then a few others here: #21165.